test/test_property.py

```diff
@@ -39,6 +39,13 @@ def test_slice_assignment(self):
         expected = [rdflib.URIRef(item) for item in c.roles]
         self.assertEqual(expected, c._properties[sbol3.SBOL_ROLE])
 
+    def test_boolean_property(self):
+        c = sbol3.Component('c1', sbol3.SBO_DNA)
+        c.boolean_attribute = sbol3.BooleanProperty(c, 'http://example.org/foo#',
```

Comment:
Just curious, why does the type uri end with `#`?

Reply:
It shouldn't.  It should be `http://example.org#foo`.  I fixed it.